src/frontend/src/components/FillForms/UserDataForm.tsx

```diff
@@ -1,337 +1,295 @@
-import React from "react";
-import history from "../../service/history";
+import React, { useState } from "react";
 import { downloadExpungementPacket } from "../../redux/search/actions";
-import { connect } from "react-redux";
-import { AppState } from "../../redux/store";
-import { AliasData } from "../RecordSearch/SearchPanel/types";
 import InvalidInputs from "../InvalidInputs";
-import moment from "moment";
+import { useAppSelector } from "../../redux/hooks";
+import { useDispatch } from "react-redux";
+import EmptyFieldsModal from "./EmptyFieldsModal";
+import { selectSearchFormValues } from "../../redux/searchFormSlice";
+import { isBlank, isDate, objectToArray } from "../../service/validators";
+import { Redirect } from "react-router-dom";
 
-interface Props {
-  aliases: AliasData[];
-  loadingExpungementPacket: boolean;
-  downloadExpungementPacket: Function;
-}
+const isPresent = (str: string) => !isBlank(str);
 
-interface State {
-  name: string;
-  dob: string;
-  mailingAddress: string;
-  phoneNumber: string;
-  city: string;
-  state: string;
-  zipCode: string;
-  invalidZipCode: boolean;
-  invalidPhone: boolean;
-  invalidBirthDate: boolean;
-}
+const errorMessagesObjectToArray = (obj: object) => objectToArray(obj);
 
-class UserDataForm extends React.Component<Props, State> {
-  private buildName = () => {
-    if (this.props.aliases.length > 0) {
-      const firstAlias = this.props.aliases[0];
-      const nameString = `${firstAlias.first_name} ${
-        firstAlias.middle_name ? firstAlias.middle_name + " " : ""
-      }${firstAlias.last_name}`;
-      if (!nameString.includes("*")) {
-        return nameString;
-      } else {
-        return "";
-      }
-    } else {
-      return "";
-    }
-  };
+const isPhoneNumber = (str: string) => {
+  //the phone RegEx accepts the following formats (123) 456-7890, 123-456-7890, 123.456.7890, 1234567890, (123)-456-7890
+  return /^(\(?\d{3}\)?[-.\s]?){2}\d{4}$/.test(str); // Check for empty string included
```

Comment:
Oh does this account for international phone numbers?

Reply:
No, just a standard 10 digit US phone number